orchestration/scripts/poll-ingest-job.py

```diff
@@ -33,12 +34,17 @@ def is_done(job_id: str) -> bool:
 def is_success(job_id: str):
     # need to spit out the lowercase strings instead of real bools, allows the workflow to know if it succeeded or not
     if check_job_status(job_id) == "succeeded":
-     return "true"
+        return "true"
     else:
-        raise Error("Ingest job ran but did not succeed, file was not ingested.")
+        raise ValueError("Ingest job ran but did not succeed, file was not ingested.")
```

Comment:
While you're in here:
```suggestion
        raise ValueError("Job ran but did not succeed.")
```
Since this is common to all types of jobs.

Reply:
Ah good point